samples/csharp_dotnetcore/14.nlp-with-dispatch/Properties/launchSettings.json

```diff
@@ -1,10 +1,11 @@
-{
+﻿{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
   "iisSettings": {
-    "windowsAuthentication": false,
-    "anonymousAuthentication": true,
+    "windowsAuthentication": false, 
+    "anonymousAuthentication": true, 
     "iisExpress": {
-      "applicationUrl": "http://localhost:3978/",
-      "sslPort": 0
+      "applicationUrl": "http://localhost:14523",
```

Comment:
We've been pretty good about having this be 3978 in all the samples, as it allows for easier re-use of a .bot file in the emulator. Consider setting to 3978.

Reply:
yes we'll do that in a pass through all the staging ... also we need README and stylecop